api_tests/lib/comit.ts

```diff
@@ -0,0 +1,59 @@
+/// HTTP API
+
+import { BtsieveForComitNodeConfig } from "./actor";
+
+export interface AcceptPayload {
+    beta_ledger_refund_identity: string;
+    alpha_ledger_redeem_identity: string;
+}
+
+export interface SwapResponse {
+    _links: any;
+    _embedded: any;
+    parameters: any;
+    role: string;
+    state: {
+        alpha_ledger: any;
+        beta_ledger: any;
+        communication: any;
+    };
+    status: string;
+}
+
+export interface Action {
+    type: string;
+    payload: any;
+}
+
+export interface SwapsResponse {
+    _link: any;
```

Comment:
:x: Should be `_links`.

You could also simply create a type called `HalResource` that contains these two properties to accurately represent, why we have these fields.

Reply:
Ok. Do you think `Swap` (few lines below) should also extend `HalResource`? Because it never contains `_embedded`. Is is it considered optional in `HalResource`?

Reply:
Yes to both questions!